### ISSUE TITLE
Is the project still under maintenance?

### ISSUE THREAD

**Issue by willylee007** [nubomediaTI/Kurento-iOS#43]
It's a nice project to learn webrtc with Kurento, and it help me a lot, THX!
Now I'm developing an iOS application with a latest Kurento version and latest OpenVidu-server and latest WebRTC Framework.
Anybody who do the same work,can contact with me.

**Comment by speedoholic** [comment]
@willylee007 how is the project going? Were you able to use the toolbox as it is or had to make some changes?

**Comment by willylee007** [comment]
This project is developed base on kurento-media-server:6.6.0, you can test it with Docker and start a kurento-room server following [docs](http://doc-kurento-room.readthedocs.io/en/latest/quick_start.html#running-the-demo)

But I developed an iOS application base on latest kms-server and latest openvidu-server and backend-server just similar to this project, so I have to update some webRTC APIs(Installed by cocoapod, without building complete webrtc-ios project from google) and singal protocols, and simplify some files. It have tested passed through iOS 9.3- 11.4 with my phones. 

I have not use the toolbox, just modify signals refer to [OpenVidu-demo](https://openvidu.io/docs/demos/openvidu-getaroom/) and this project. You can debug these signals with Internet Tool in Chrome Debug mode.


I will open source this project later , and others can run or test this project with Docker locally.

**Comment by speedoholic** [comment]
@willylee007 thanks for your prompt response. Will try to setup a room using Ubuntu on docker. So after configuring the server on `ws://localhost:8888/kurento`, should the websocket URI be `wss://localhost:8443/room` ?

**Comment by willylee007** [comment]
Yes. This project may have some problems on iOS 10+, but is OK on iOS9.

**Comment by speedoholic** [comment]
@willylee007 what about the web? Tried it on chrome but doesn't seem to get or publish the stream

**Comment by willylee007** [comment]
It works on web.

Download kurento-media-server:6.6.0 image in Docker, then run follow command 

`
docker run -p 8888:8888 --rm b7eadc9f1116
`

Remember change "b7eadc9f1116" with your own image id.

Then compile kurento-room and start service:

`
$ git clone https://github.com/Kurento/kurento-room.git
`
`
$ cd kurento-room
`
`
$ git checkout $(git describe --abbrev=0 --tags)
`
`
$ cd kurento-room-demo
`
`
$ mvn compile exec:java
`

Then open chrome and connect: https://localhost:8443, not http://localhost:8443

You can debug signals with Network Tool in Chrome:
<img width="753" alt="qq20180703-220122 2x" src="https://user-images.githubusercontent.com/16423055/42224628-29af90b6-7f0d-11e8-984e-e7308cbd8dd2.png">

That works for me.

More details refer to [Running the demo](http://doc-kurento-room.readthedocs.io/en/latest/quick_start.html#running-the-demo)

Good luck~
